executor/src/core.rs

```diff
@@ -150,27 +174,46 @@ where
             {
                 // Execute the transaction
                 let result = self
-                    .execute_transaction(transaction.clone(), total_transactions, total_batches)
+                    .execute_transaction(
+                        consensus_output,
+                        transaction.clone(),
+                        total_transactions,
+                        total_batches,
+                    )
                     .await;
 
-                // Output the result (eg. to notify the end-user);
-                let output = (result.clone(), transaction);
-                if self.tx_output.send(output).await.is_err() {
-                    debug!("No users listening for transaction execution");
-                }
+                let (bail, result) = match result {
+                    Ok((outcome, committee)) => {
+                        if let Some(_committee) = committee {
+                            // todo!();
```

Comment:
Could we leave a note about what's left to do here? Perhaps clarify the conditions in which we would return `bail`? Log the epoch change, which may condition the validity of the sequencing output by consensus?